src/XMakeBuildEngine/Utilities/Utilities.cs

```diff
@@ -459,7 +459,7 @@ internal static PropertyDictionary<ProjectPropertyInstance> GetEnvironmentProper
             // environment or as a global property.  
 
 #if !FEATURE_INSTALLED_MSBUILD
```

Comment:
I'd like to get rid of all of the feature stuff and only have NETCORE, MONO or FULL (if possible).